.changelog/9198.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+server: All metrics should be present and available to prometheus scrapers when Consul starts. If any non-deprecated metrics are missing please submit an issue with its name.
```

Comment:
very minor: I know we aren't very precise with our categories, but I wonder if `agent:` would be more appropriate than `server:`.

Reply:
Makes sense - client agents are affected too